src/Take.Blip.Builder/Utils/SalesForce/Models/SalesForceRoutes.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Take.Blip.Builder.Utils.SalesForce.Models
+{
+    /// <summary>
+    /// Sales force routes
+    /// </summary>
+    public static class SalesForceRoutes
```

Comment:
Maybe chante this to SalesForceUriTemplates ( to follow the rest of the iris pattern )